dockerfiles/mesh-bootstrap/Dockerfile

```diff
@@ -13,7 +13,7 @@ WORKDIR /go/src/github.com/0xProject/0x-mesh
 
 ADD . ./
 
-RUN make deps-go-no-lockfile
+RUN make gobin
```

Comment:
`gobin` is used for installing tools that we run from the command line (specifically `gowasmexec`, which is used in the tests). I don't actually think we need it here. We can just skip this step.

```suggestion
```